Recipe Search/Recipe Search/Controllers/ResultViewController.swift

```diff
@@ -90,3 +110,11 @@ extension ResultViewController: UICollectionViewDelegate {
         }
     }
 }
+
+extension ResultViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView,
+                        layout collectionViewLayout: UICollectionViewLayout,
+                        referenceSizeForFooterInSection section: Int) -> CGSize {
+        return isFetching ? CGSize(width: view.frame.width, height: 100) : CGSize(width: 0, height: 0)
```

Comment:
hoặc e có thể sử dụng như này cho gọn 
```suggestion
        return isFetching ? CGSize(width: view.frame.width, height: 100) : .zero
```